editor/modes/text-editor/index.js

```diff
@@ -11,12 +11,32 @@ import './style.scss';
 
 function TextEditor( { blocks, onChange } ) {
 	return (
-		<Textarea
-			defaultValue={ wp.blocks.serialize( blocks ) }
-			onBlur={ ( event ) => onChange( event.target.value ) }
-			className="editor-text-editor"
-			useCacheForDOMMeasurements
-		/>
+		<div>
+			<header className="editor-text-editor__formatting">
+				<div>
```

Comment:
Can we lose the `div` and apply the padding to the `.editor-text-editor__formatting` directly? In any case we should try to avoid applying styles via element selectors, considering there's a non-zero likelihood one of these buttons may eventually contain their own `div` descendants.